src/Shell.php

```diff
@@ -1201,13 +1223,14 @@ public function getLastExecSuccess(): bool
      */
     public function formatException(\Exception $e): string
     {
-        $message = $e->getMessage();
-        if (!$e instanceof PsyException) {
-            if ($message === '') {
-                $message = \get_class($e);
-            } else {
-                $message = \sprintf('%s with message \'%s\'', \get_class($e), $message);
-            }
+        if ($e instanceof BreakException) {
+            return sprintf("  <fg=white;bg=blue;options=bold> INFO </> %s.", $e->getRawMessage());
```

Comment:
i'd prefer to use named styles rather than inline styles… for all the reasons classes are better than inline styles in HTML :P

Reply:
Besides named style, 2 little details from me:
```suggestion
            return sprintf('  <fg=white;bg=blue;options=bold> INFO </> %s.', rtrim($e->getRawMessage(), '.'));
```

* Use single quotes, since interpolation is not required
* Ensure there's single `.` at the end (or just remove dot from the message)